@xen-orchestra/web-core/lib/components/tab/tab-list.composable.ts

```diff
@@ -22,7 +22,7 @@ export function useTabList<TName extends string>(names: TName[], initialTab?: Ma
               event.preventDefault()
               activate(name)
             },
-            active: isActive(name),
+            selected: isActive(name),
```

Comment:
`selected`: `isSelected(name)` maybe? In order to sync name. (or `isCurrent()` if the proposition is accepted)